### PR TITLE
RDCC-1746 : reduce CVE in suppressions.xml.

### PR DESCRIPTION
**Pull request by dharmendra8186**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-1746

### Change description ###
Mitigate Profile Sync vulnerabilities

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -1,199 +1,18 @@
...
-        <packageUrl regex="true">^pkg:maven/org\.postgresql/postgresql@.*$</packageUrl>
-        <cpe>cpe:/a:postgresql:postgresql</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: postgresql-42.2.9.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.postgresql/postgresql@.*$</packageUrl>
-        <cpe>cpe:/a:postgresql:postgresql_jdbc_driver</cpe>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.sling/org\.apache\.sling\.javax\.activation@.*$</packageUrl>
```
Comment (abhijitdiwan) [inline config/owasp/suppressions.xml:14]: we can remove this. use testCompile (group: 'io.rest-assured

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -1,199 +1,18 @@
...
+        <cve>CVE-2016-5394</cve>
```
Comment (KOTLAPRA23) [inline config/owasp/suppressions.xml:15]: is this required here?

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (dharmendra8186) [reply config/owasp/suppressions.xml:14]: We can't remove this, as it's giving me compilation error.Although, I tried  to upgrade version of rest assure jar and still getting test case failure issue.Hence, I have to suppress it.

File: config/owasp/suppressions.xml (hunk 2 above)
Comment (dharmendra8186) [reply config/owasp/suppressions.xml:15]: yeah, it's required.

**Pull request merged**